Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.10.3-slim-buster as base
+RUN apt-get update
+COPY . /opt/
+WORKDIR /opt
+RUN pip install -r requirements.txt
+
+FROM base as production
+EXPOSE 80
+ENTRYPOINT ["sh", "/opt/gunicorn.sh"]
```

Comment:
If you make the files executable (e.g. RUN chmod +x /opt/gunicorn.sh) , you could run them directly: `ENTRYPOINT [ "/opt/gunicorn.sh" ]`

Also, you don't need an absolute path. You are already working in the /opt folder, so you can just write gunicorn.sh or `./gunicorn.sh` to mean "in the current folder".

Otherwise you have to keep WORKDIR and this ENTRYPOINT in sync